hack/coreapi-server.sh

```diff
@@ -12,13 +12,15 @@ exec mod_wsgi-express start-server \
                       --port 5000 \
                       --user coreapi \
                       --group coreapi \
-                      --processes 4 \
-                      --threads 10 \
+                      --processes 1 \
```

Comment:
Let's try load testing with multi-process multi-threads here.

Reply:
@miteshvp need to try and check. Mercator CLI runs on a thread and having a thread to run another thread basically halts the processing and results in gateway timeout. I will do my research and get back on this.

Reply:
@miteshvp in order to run stack analysis with the verified performance results, we need this configuration in coreapi-server.sh